stages/org.osbuild.chattr

```diff
@@ -14,24 +14,42 @@ import osbuild.api
 SCHEMA_2 = r"""
 "options": {
   "additionalProperties": false,
-  "required": ["path", "immutable"],
   "properties": {
-    "immutable": {
-      "type": "boolean",
-      "description": "Make the file/directory immutable",
-      "default": true
-    },
-    "path": {
+    "items": {
       "oneOf": [
         {
-          "type": "string",
-          "description": "The destination, if a mount",
-          "pattern": "^mount:\/\/[^\/]+\/"
+          "type": "object",
+          "additionalProperties": false,
+          "patternProperties": {
+            "^mount:\/\/[^\/]+\/": {
```

Comment:
I was thinking originally that we should just copy the regex here verbatim from the chmod or chown stages but it looks like those don't really account for being able to operate on the tree or on a mount..

in the very least maybe we could just update the regex here to support both so we don't have to copy/paste the rest of it?

Would this work?

```suggestion
            "(^mount:\/\/[^\/]+\/|^tree:\/\/\/": {
```